src/coins.h

```diff
@@ -321,8 +339,9 @@ struct CNullifiersCacheEntry
     CNullifiersCacheEntry() : entered(false), flags(0) {}
 };
 
-typedef boost::unordered_map<uint256, CCoinsCacheEntry, CCoinsKeyHasher> CCoinsMap;
-typedef boost::unordered_map<uint256, CAnchorsCacheEntry, CCoinsKeyHasher> CAnchorsMap;
+typedef boost::unordered_map<uint256, CCoinsCacheEntry, CCoinsKeyHasher>      CCoinsMap;
+typedef boost::unordered_map<uint256, CSidechainsCacheEntry, ObjectHasher>    CSidechainsMap;
```

Comment:
Maybe the same type of hasher can be used for all maps

Reply:
I changed hasher to CCoinsKeyHasher and verified code correctness is not affected.